lib/galaxy/tools/error_reports/plugins/slack.py

```diff
@@ -0,0 +1,105 @@
+"""The module describes the ``slack`` error plugin plugin."""
+import logging
+import uuid
+
+import requests
+
+from galaxy import web
+from galaxy.util import string_as_bool
+from .base_git import BaseGitPlugin
+
+log = logging.getLogger(__name__)
+
+
+class SlackPlugin(BaseGitPlugin):
+    """Send error report to Sentry."""
+
+    plugin_type = "slack"
+
+    def __init__(self, **kwargs):
+        self.app = kwargs["app"]
+        self.redact_user_details_in_bugreport = self.app.config.redact_user_details_in_bugreport
+        self.verbose = string_as_bool(kwargs.get("verbose", False))
+        self.user_submission = string_as_bool(kwargs.get("user_submission", False))
+        self.webhook_url = kwargs.get("webhook_url", "https://localhost/")
+
+    def _append_issue(self, *args, **kwargs):
+        pass
+
+    def _create_issue(self, *args, **kwargs):
+        pass
+
+    def _fill_issue_cache(self, *args, **kwargs):
+        pass
+
+    def submit_report(self, dataset, job, tool, **kwargs):
+        history_id_encoded = self.app.security.encode_id(dataset.history_id)
+        history_view_link = web.url_for("/histories/view", id=history_id_encoded, qualified=True)
+        error_report_id = str(uuid.uuid4())[0:13]
+        title = self._generate_error_title(job)
+
+        blocks = {
+            "blocks": [
+                {
+                    "type": "section",
+                    "text": {
+                        "type": "mrkdwn",
```

Comment:
just checking, it really is `mrkdown` ?

Reply:
ok, googled that and it's a thing

Reply:
[yes, it is really `mrkdwn` missing all vowels.](https://api.slack.com/reference/surfaces/formatting). I don't know what slack has against vowels but it looks obviously wrong to me every time I see it.